regression/pages/whitelabel/const.py

```diff
@@ -8,15 +8,16 @@
 AUTH_USERNAME = os.environ['BASIC_AUTH_USER']
 AUTH_PASSWORD = os.environ['BASIC_AUTH_PASSWORD']
 
-# Get API Access token
+# Global password
+PASSWORD = os.environ.get('GLOBAL_PASSWORD')
 
-ACCESS_TOKEN = os.environ['ACCESS_TOKEN']
+# Client information for access token
+OAUTH_CLIENT_ID = os.environ.get('OAUTH_CLIENT_ID')
+OAUTH_CLIENT_SECRET = os.environ.get('OAUTH_CLIENT_SECRET')
 
-# Select the Org for which to run the tests, Default is MITProfessionalX
-ORG = os.getenv('ORG', 'MITProfessionalX')
+# Select the Org for which to run the tests, Default is HarvardMedGlobalAcademy
+ORG = os.environ.get('ORG', 'HarvardMedGlobalAcademy')
```

Comment:
Why we are setting `HarvardMedGlobalAcademy` as default organization? as previously it was `MITProfessionalX`. I think we can either change the placeholder name or there should be no default.

Reply:
business priority, right now only HarvardMedGlobal tests are active so it makes more sense to have it as default value

Reply:
we can modify it like:
```python
PRIORITIZED_ORG_FOR_TESTING = 'HarvardMedGlobalAcademy'
ORG = os.environ.get('ORG', PRIORITIZED_ORG_FOR_TESTING)
```